multiapps-controller-process/src/main/resources/org/cloudfoundry/multiapps/controller/process/backup-existing-app.bpmn

```diff
@@ -10,9 +10,17 @@
     <serviceTask id="stopApplicationUndeploymentTask" name="Stop Application Undeployment Step" flowable:async="true" flowable:delegateExpression="${stopApplicationUndeploymentStep}"></serviceTask>
     <endEvent id="endEvent"></endEvent>
     <exclusiveGateway id="hasStepWithHooksCompleted" default="reExecuteStepFlow"></exclusiveGateway>
-    <callActivity id="executeHooksCallActivity" name="Execute Hooks Call Activity" flowable:async="true" calledElement="${hookProcessGetter.get(hookForExecution, execution)}" flowable:calledElementType="key" flowable:inheritVariables="true" flowable:completeAsync="true" flowable:fallbackToDefaultTenant="false">
+    <callActivity id="executeHooksCallActivity" name="Execute Hooks Call Activity" flowable:async="true" calledElement="${hookProcessGetter.get(hookForExecution, execution)}" flowable:calledElementType="key" flowable:inheritVariables="false" flowable:completeAsync="true" flowable:fallbackToDefaultTenant="false">
+    <extensionElements>
+      <flowable:in source="appToProcess" target="appToProcess"></flowable:in>
+      <flowable:in source="namespace" target="namespace"></flowable:in>
+      <flowable:in source="mtaDeploymentDescriptor" target="mtaDeploymentDescriptor"></flowable:in>
+      <flowable:in source="__SPACE_ID" target="__SPACE_ID"></flowable:in>
+      <flowable:in source="hookForExecution" target="hookForExecution"></flowable:in>
+    </extensionElements>
```

Comment:
Is the variable tasksToExecute needed? I see it used in PrepareToExecuteTasksStep

Reply:
startedTask, correlationId?